### PR TITLE
Add `Requests::has_capability()` which can be used to determine if SSL is available.

### PR DESCRIPTION
**Pull request by dd32**
Add `Requests::has_capability()` which can be used to determine if SSL is available.

An example of it's usage: `Requests::has_capability( array( 'ssl' => true ) );`

See #250

### CODE REVIEW

File: library/Requests.php [hunk 1]
```diff
@@ -217,6 +217,21 @@ protected static function get_transport($capabilities = array()) {
...
 
+	/**
+	 * Checks to see if we have a transport for the capabilities requested.
+	 *
+	 * @return bool
+	 */
+	public static function has_capability($capabilities = array()) {
+		try {
+			$transport = self::get_transport($capabilities);
+		} catch(Requests_Exception $e) {
```
Comment (rmccue) [inline library/Requests.php]: Needs a space after `catch`

**Comment by schlessera** [comment]
This PR was closed inadvertently when we changed the stable branch from `master` to `stable`. GitHub was supposed to smoothly adapt the PRs to point to the new branch, but this seems to have failed for the cases where the author's branch/fork of the PR is not available anymore.

**Comment by jrfnl** [comment]
Recreated as #492

**Comment by jrfnl** [comment]
PR #492 has now been merged, so this will be included in the 2.0.0 release.

**Pull request closed without merging**